### ISSUE TITLE
Using "window" as a local variable inside a closure

### ISSUE THREAD

**Issue by Irrelon** [nodeca/pako#60]
Hey ya,

I've noticed that you declare "window" and use it throughout the library but this interferes with the externally named global "window" when instrumenting code for coverage stats. I replaced "window" in my cloned version of the repo with the variable name "sWindow" (although obviously any name would do) and the library still worked but didn't interfere with code coverage either.

FYI I'm using the library via browserify to compile all the project JS and modules to a single JS file for browser consumption. After that I have a qunit process running in grunt that goes over the whole thing and checks against unit tests and code coverage.

**Comment by puzrin** [comment]
I understand your problem and agree in general, but would like to see diff first. Could you show it?

**Comment by Irrelon** [comment]
All I did was search/replace the word "window" in the lib folder with the word "sWindow" and made sure I ticked "Whole words only" in the search/replace dialog in the JetBrains IDE I'm using (PHPStorm).

I replaced all instances automatically.

I didn't commit changes afterwards or push to a forked repo so I don't have a diff for you, but if you like I can do it again and do a pull request... my main reason for not doing this though is that I assumed the PR would be massively out of date by the time someone got around to viewing it and by then many of the affected files would have been changed leading to merge conflicts all over the place :)

Let me know if you want me to do a pull request for it though and I'll put it together for you.
